CONTRIBUTING.md

```diff
@@ -111,32 +111,6 @@ If you are going to make significant contribution(s) to the track, you might fin
 
 The easiest way to achieve this is simply use the `bin/journey-test.sh` script.  However, you may want to perform other tests, depending on what you are doing.  You can do so by duplicating the setup performed by the `bin/journey-test.sh` script.
 
-### Tip: `gradle clean` before `exercism fetch`
```

Comment:
I don't think we need this section since by the sounds of it it's related to how the journey-test used to work 🙂